src/Judge/NTS.Judge.MAUI/NTS.Judge.MAUI.csproj

```diff
@@ -12,10 +12,10 @@
     <LangVersion>preview</LangVersion>
     
     <!-- Display name -->
-    <ApplicationTitle>EMS.Judge</ApplicationTitle>
+    <ApplicationTitle>NTS Judge</ApplicationTitle>
 
     <!-- App Identifier -->
-    <ApplicationId>com.es.ems.judge</ApplicationId>
+    <ApplicationId>com.nts.judge</ApplicationId>
```

Comment:
Let's keep this `com.ems.judge`, since it will otherwise messup the app deployment to stores